<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qpid::AboutDialog</class>
 <widget class="QDialog" name="Qpid::AboutDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>212</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Voice Easy</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background:  #ebf4fa;
	border: 1px solid #3d89bf;
}

#frameCaption {
	border-image: url(:/images/ui/top_caption.png);
}

#pushButtonClose {
	border-image: url(:/images/ui/close.png);
}

#pushButtonClose:hover {
	border-image: url(:/images/ui/close_hover.png);
}

#labelTitle {
	color: rgb(255,255,255);
	padding-left: 3px;
	font-size: 15px;
	font-weight: bold;
}

#versionsLabel {
	font-size: 15px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frameCaption">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTitle">
        <property name="text">
         <string>Voice Easy</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="sysPanelContainer">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="sysPanel" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButtonClose">
              <property name="minimumSize">
               <size>
                <width>35</width>
                <height>30</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>9</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="aboutPanel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>16</number>
      </property>
      <property name="leftMargin">
       <number>39</number>
      </property>
      <item>
       <widget class="QLabel" name="appIcon">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="versionsLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="phone.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>Qpid::AboutDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
